test/runnable/iasm64.d

```diff
@@ -3727,7 +3731,11 @@ void test50()
         int     3       ;
         int     0x67    ;
         //into          ;
-L10:    iret            ;
+        iret            ;
+        iretq           ;
+L10:    nop             ; // put instructions above this or L10 changes
```

Comment:
You will have to add two `nop`s to simulate the length of the `iret` instruction while avoiding changing all jumps.

Reply:
Oh man, wrong again. Sorry :|